OPERATOR.rst

```diff
@@ -156,28 +156,55 @@ label from blocking PR. Last, remove the blocking relationship.
 Upgrading GitLab
 ^^^^^^^^^^^^^^^^
 
-Occasionally it falls on the operator to upgrade the Azul GitLab instance. If
-the current major version is ``n`` and the latest available major version is
-greater than ``n+1`` (i.e. upgrading directly to the latest version would skip
-one or more major versions) then multiple successive upgrades must be made, such
-that no upgrade skips a major version. For example, if the current version is
-13.x.y and the latest available version is 15.x.y, then one would first upgrade
-to 14.x.y and then repeat the process to upgrade to 15.x.y.
-
-Before any changes are applied, stop the instance (do not terminate) and create
-a snapshot of its EBS volume. Edit ``terraform/gitlab/gitlab.tf.json.template.py``,
-updating the versions of the docker images for ``gitlab-ce`` and
-``gitlab-runner``. Then run::
+Operators must check for updates to GitLab on a daily basis. Updates to Gitlab
+can be quickly checked at the ``/help`` endpoint for `GitLab dev`_ and
+`GitLab prod`_. If there are updates available, find the new release of the
+`GitLab Docker image`_ and update the `Gitlab Terraform`_ file. When updating
+the GitLab instance, check if there are applicable updates to the
+`GitLab runner image`_. Before starting the update process check the
+`GitLab release notes`_ for upgrading instructions. When upgrading across
+multiple GitLab versions, follow the prescribed GitLab `upgrade path`_.
+
+Before any changes are applied, source the appropriate ``gitlab`` environment
+file, fetch remote repositories, then create a new branch called
+``gitlab/<yyy-mm-dd>``  with ``github/develop`` as its start-point.
+
+Stop the GitLab EC2 instance (do not terminate), and create a snapshot of its
+EBS volume. To easily create a snapshot run::
+
+	python scripts/create_gitlab_snapshot.py
```

Comment:
With the component selected or the corresponding main deployment?

Reply:
I'm confused about the term `component` here. I updated the block to specify that `_select gitlab.dev` should be ran before the script is called.

Because both the `prod` and `dev` instance of gitlab have the same name (`azul-gitlab`),  If the operator has `dev`, `sandbox`, or `dev.gitlab` selected, the snapshots for the dev instance are created.

I added this fixup https://github.com/DataBiosphere/azul/issues/3820#issuecomment-1029510044

Reply:
https://github.com/DataBiosphere/azul#9-continuous-deployment-and-integration defines that term. We'll stick with the current names. I closed #3820.

Reply:
I added some prerequisite commands that the operator has to follow. 
https://github.com/DataBiosphere/azul/pull/3627#discussion_r806309129

The last thing they do it `_select` the environment file for the applicable GitLab subcomponent.
I updated the condition of the require within the script as well, so a main deployment cannot be selected.